File: frontend/fix-dashboard-new.js

const fs = require('fs');
const path = require('path');

const filePath = path.join(__dirname, 'src', 'pages', 'Dashboard_New.jsx');
let content = fs.readFileSync(filePath, 'utf8');

// Fix imports
content = content.replace('from "../context/LanguageContext"', 'from "../contexts/LanguageContext"');
content = content.replace('from "../components/layout/AppLayout"', 'from "../components/layouts/AppLayout"');

// Fix useLanguage usage
content = content.replace('const { lang, setLang } = useLanguage();', 'const { language, setLanguage } = useLanguage();');
content = content.replace('langs.indexOf(lang)', 'langs.indexOf(language)');
content = content.replace('setLang(nextLang)', 'setLanguage(nextLang)');
content = content.replace('currentLang={lang}', 'currentLang={language}');

fs.writeFileSync(filePath, content, 'utf8');
console.log('✅ Fixed Dashboard_New.jsx!');

File: fix_dashboard_new.py

with open(r'd:\agrifuturedo\frontend\src\pages\Dashboard_New.jsx', 'r', encoding='utf-8') as f:
    content = f.read()

# Fix imports
content = content.replace(r'from "../context/LanguageContext"', r'from "../contexts/LanguageContext"')
content = content.replace(r'from "../components/layout/AppLayout"', r'from "../components/layouts/AppLayout"')

# Fix useLanguage usage
content = content.replace('const { lang, setLang } = useLanguage();', 'const { language, setLanguage } = useLanguage();')
content = content.replace('langs.indexOf(lang)', 'langs.indexOf(language)')
content = content.replace('setLang(nextLang)', 'setLanguage(nextLang)')
content = content.replace('currentLang={lang}', 'currentLang={language}')

with open(r'd:\agrifuturedo\frontend\src\pages\Dashboard_New.jsx', 'w', encoding='utf-8') as f:
    f.write(content)

print('✅ Fixed Dashboard_New.jsx!')
